tsconfig.json

```diff
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "target": "es2015",
     "jsx": "react",
+    "module": "esnext",
```

Comment:
This line was needed, again, to support our polyfill CSS. I kept getting errors about not being able to import a module.